beets/util/confit.py

```diff
@@ -248,7 +248,11 @@ def set_args(self, namespace):
         like argparse or optparse, onto this view's value.
         """
         args = {}
-        for key, value in namespace.__dict__.items():
+        if isinstance(namespace, dict):
+            items = namespace.items()
+        else:
+            items = namespace.__dict__.items()
+        for key, value in items:
```

Comment:
this is the patch.  seems to let you use it like `foo.bar` as well as `foo['bar']`.